### ISSUE TITLE
Rule proposal: AX_ARIA_04

### ISSUE THREAD

**Issue by lencioni** [evcohen/eslint-plugin-jsx-a11y#13]
I think that this would make a really good linter rule. https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_04

ARIA state and property values must be valid

The values of ARIA states and properties must be of the correct type.

Bad:

```html
<!-- Bad: the aria-hidden state is of type true/false -->
<span aria-hidden="yes">foo</span>
```

Good:

```html
<!-- Good: the aria-hidden state is of type true/false -->
<span aria-hidden="true">foo</span>
```
